sass/custom/_paulus.scss

```diff
@@ -309,19 +308,23 @@ article.post, article.page, article.listing {
 
   h2 {
     font-size: 1.5em;
-    margin-top: 2em;
+    margin: 1.5em 0 1rem;
```

Comment:
This applies to ALL headers. Let's do a re-design of the website in another PR please.